mapswipe_workers/mapswipe_workers/project_types/change_detection/tile_functions.py

```diff
@@ -57,8 +56,11 @@ def lat_long_zoom_to_pixel_coords(lat, lon, zoom):
     p = Point()
     sinLat = math.sin(lat * math.pi / 180.0)
     x = ((lon + 180) / 360) * 256 * math.pow(2, zoom)
-    y = (0.5 - math.log((1 + sinLat) / (1 - sinLat)) /
-         (4 * math.pi)) * 256 * math.pow(2, zoom)
+    y = (
```

Comment:
it looks like this file or most of it is duplicated between the different project types. Do you actually need that? Is there a way you can refactor these functions/classes into a file that you can then import in the various project types?

Reply:
Totally true. I moved the tile functions to `utils`.